pkgs/applications/audio/traverso/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit, cmake
+, alsaLib, fftw, flac, lame, libjack2, libmad, libpulseaudio
+, libsamplerate, libsndfile, libvorbis, portaudio, qtbase, wavpack
+}:
+stdenv.mkDerivation rec {
+  name = "traverso-${version}";
+  version = "0.49.5";
+
+  src = fetchgit {
+    url = "https://git.savannah.gnu.org/git/traverso.git";
+    rev = "2e215feaa9aebe104658c14d1820abdece7fb287";
```

Comment:
This git repo is a mess... :/  That rev isn't actually v0.49.5.  CMakeLists even set:
```cmake
SET (VERSION_MAJOR "0")
SET (VERSION_MINOR "49")
SET (VERSION_PATCH "1")
```
... and there is no proper tag for the version.

So it would be better to `fetchurl` the source tarball at http://traverso-daw.org/traverso-0.49.5.tar.gz instead

Reply:
haha aye it is. Mostly adding for some software archeology.

I'll switch it to fetchurl . That has the 49.5 set in CMakeLists.